mode/php/php.js

```diff
@@ -53,7 +53,7 @@
           [["]", null]]
         ], closing, escapes);
       }
-      if (stream.match(/\-\>\w/, false)) {
+      if (stream.match(/^->\w/, false)) {
```

Comment:
No need to escape `>` or `-`, except they are in a special position, which they are not in this case.